packages/compass-signtool/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@mongodb-js/compass-signtool",
+  "description": "Signtool to sign Compass windows artifacts",
```

Comment:
from the sync we had, i understood we want do have this also for signing locally for linux machines, and not only for Compass, ultimately have it in devtools-shared, can we name this differently and adjust the description?